src/components/ProductsFeatured/index.tsx

```diff
@@ -0,0 +1,56 @@
+import './scss/index.scss';
```

Comment:
In other files we stick to an importing file convention: import npm packages first, then local js files and scss styles at last.

Reply:
as far as I can see in most .tsx files styles are imported after js files. Anyway, let's keep it consistent throughout the project :)